### DISCUSSION TITLE
Please consider removing scroll animations

### DISCUSSION THREAD

**Discussion by brianpeiris** [mdn/mdn-community#107] [Platform]
### Summary

MDN is a reference site. I visit it almost everyday to get to information I need. Scroll animations hinder my efficiency. I should not have to use an accessibility feature (reduced motion) to get simple things done quickly.

As an example, the list of HTTP status codes is a document I frequently come back to for reference. Every time I visit it to jog my memory, I have to wait an additional half a second for the scroll animation to get me to a section of the document. This is pretty frustrating when I know exactly what I want. The scroll animation is not a UX feature, and it is not helpful at all. It does not add to my delight. I'm here for information, not entertainment.

### URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status 

<details>

### Reproduction steps

1. Visit this page almost daily https://developer.mozilla.org/en-US/docs/Web/HTTP/Status 
2. Click on one of the status code categories
3. Wait for the scroll before you can get to the info.

### Expected behavior

MDN helps me get to the information I want as quickly as possible.

### Actual behavior

Scroll animations slow me down on a daily basis.

### Device

Desktop

### Browser

Firefox

### Browser version

Stable

### Operating system

Windows

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).

</details>

**Comment by jaas666** [comment]
I agree. Looks cool but I don't think we need it here. 

Getting the info as fast as possible is more important than style in this case.

**Comment by OnkarRuikar** [comment]
Also, fast scrolling gives dizzy feeling.

https://user-images.githubusercontent.com/87750369/170683332-88933881-9380-4a51-8553-9cf7660f6be9.mov

**Comment by schalkneethling** [comment]
I have to say I agree with this. We do disable it if a user has set `prefers-reduced-motion` but, it does feel like we should disable this in general.
